presto-docs/src/main/sphinx/connector/hana.rst

```diff
@@ -0,0 +1,100 @@
+if you name the property file ``sales.properties``, Presto will create a
+catalog named ``sales`` using the configured connector.
+
+General Configuration Properties
+---------------------------------
+
+================================================== ==================================================================== ===========
+Property Name                                      Description                                                          Default
+================================================== ==================================================================== ===========
+``user-credential-name``                           Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user name. See ``extraCredentials`` in `Parameter Reference
+                                                   <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``password-credential-name``                       Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user password. See ``extraCredentials`` in `Parameter
+                                                   Reference <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``case-insensitive-name-matching``                 Match dataset and table names case-insensitively.                    ``false``
+
+``case-insensitive-name-matching.cache-ttl``       Duration for which remote dataset and table names will be
+                                                   cached. Set to ``0ms`` to disable the cache.                         ``1m``
+================================================== ==================================================================== ===========
+
+Querying HANA
+-------------------
+
+The HANA connector provides access to all schemas visible to the specified user in the configured database.
+For the following examples, assume the HANA catalog is ``hana``.
+
+You can see the available schemas by running ``SHOW SCHEMAS``::
+
+    SHOW SCHEMAS FROM hana;
+
+If you have a schema named ``web``, you can view the tables
+in this schema by running ``SHOW TABLES``::
+
+    SHOW TABLES FROM hana.web;
+
+You can see a list of the columns in the ``clicks`` table in the ``web`` database
+using either of the following::
+
+    DESCRIBE hana.web.clicks;
+    SHOW COLUMNS FROM hana.web.clicks;
+
+Finally, you can query the ``clicks`` table in the ``web`` schema::
+
+    SELECT * FROM hana.web.clicks;
+
+If you used a different name for your catalog properties file, use
+that catalog name instead of ``hana`` in the above examples.
+
+HANA Connector Limitations
+--------------------------------
+
+The following SQL statements are not yet supported:
```

Comment:
```suggestion
The following SQL statements are not supported:
```

In general, documentation should describe the current state of the product, and avoid future predictions.